internal/commands/server/create.go

```diff
@@ -369,6 +372,13 @@ func (s *createCommand) ExecuteWithoutArguments(exec commands.Executor) (output.
 	logline.SetMessage(fmt.Sprintf("%s: request sent", msg))
 	logline.MarkDone()
 
+	if s.wait.Value() {
+		logline := exec.NewLogEntry(msg)
```

Comment:
Is it better to create new log line or update the existing one?

Reply:
I guess it would be nice if only that one line would update because it's more of an status display rather that event logging but I don't have that strong opinion on this one.

Reply:
I'll test how this would work if existing logline was updated instead of creating a new one.

Would it be better to log warning about failed `--wait` and details of created server, instead of only wait timeout error, if server does not reach given state within five minutes?

Reply:
I'll keep this two message approach for now. On timeout error, it is easier to see that error comes from `--wait` if the logline from main command is still visible. This is something we could revisit later, though.

Also, seems that currently it is not possible to easily output both error and output from command, so printing both timeout error and server details would require further refactoring.

Reply:
Sorry for late reply. Somehow I missed your question last week :)

I think that it would better to just show warning if wait fails.

Reply:
Added new commit to use single logline and output only warning on `--wait` failure. This way the user sees the UUID and IP of created server even if `--wait` timeouts, which should be unlikely.